outputs.tf

```diff
@@ -1,8 +0,0 @@
-##############################################################################
```

Comment:
I've just noticed that there are no outputs, which violates terraform module guidelines. We probably need some type of output here.

One suggestion would be to output some detail about groups and accounts (two outputs), where we maybe output the name and CRN? Something like that? Otherwise this DA will not be able to be a reference for anything else in a project stack.